lib/src/commands/LayoutTreeParser.test.ts

```diff
@@ -174,7 +197,7 @@ const optionsSplitView: OptionsSplitView = {
   displayMode: 'auto',
   primaryEdge: 'leading',
   minWidth: 150,
-  maxWidth: 300,
```

Comment:
This change made me so happy and gave me energy to continue reviewing this looooong pr 😅

Reply:
Glad you like it xD